File: Feitos em C/2717.c
#include <stdio.h>
int main() {
	int n,a,b;
	scanf("%d",&n);
	scanf("%d",&a);
	scanf("%d",&b);
	if(n>=a+b){
		printf("Farei hoje!\n");
	}
	else{
		printf("Deixa para amanha!\n");
	}
	return 0;
}

File: Feitos em Java/2717.java
import java.io.IOException;
import java.util.Scanner;
public class Main {
    public static void main(String[] args) throws IOException {
        Scanner leitor = new Scanner(System.in);
        int n = leitor.nextInt();
        int a = leitor.nextInt();
        int b = leitor.nextInt();
        if(n>=a+b){
            System.out.println("Farei hoje!");
	}
	else{
            System.out.println("Deixa para amanha!");
	}
    }
}